modules/ignite-cli-2/src/main/java/org/apache/ignite/cli/core/call/CallExecutionPipeline.java

```diff
@@ -71,49 +62,75 @@ public void runPipeline() {
         output.println(decoratedOutput.toTerminalString());
     }
 
-    /** Builder for {@link DefaultCallExecutionPipeline}. */
-    public static class DefaultCommandExecutionPipelineBuilder<I extends CallInput, T> {
+    /**
+     * Builder helper method.
+     *
+     * @return builder for {@link CallExecutionPipeline}.
+     * */
+    public static <I extends CallInput, T> CallExecutionPipelineBuilder<I, T> builder(
+            Call<I, T> call) {
+        return new CallExecutionPipelineBuilder<>(call);
+    }
+
+    /**
+     * Helper method for running picocli commands.
+     * */
+    public static <I extends CallInput, T> void runPipeline(
+            Call<I, T> call,
+            CommandSpec spec,
```

Comment:
CallExecutionPipeline should not depend on CommandSpec and any other picocli classes.